packages/data-schema/src/ModelType.ts

```diff
@@ -224,6 +227,16 @@ export type ModelType<
 > = Omit<
   {
     [brandSymbol]: typeof brandName;
+
+    /**
+     * Defines single-field or composite identifiers
+     *
+     * @param identifier A list of field names used as identifiers for the data model
+     * @returns A ModelType instance with updated identifiers
+     *
+     * @example
+     * model().identifier(['name', 'age'])
```

Comment:
```suggestion
     * a.model({
     *   name: a.string(),
     *   age: a.string(),
     * }).identifier(['name', 'age'])
```

for examples like this it'd be helpful to see the `model()` populated (assuming this multi-line examples renders okay on hover)

Reply:
Good point, I have added an example and also a comment noting the fields used as identifiers have to be marked `required`.